README.md

```diff
@@ -1,43 +1,45 @@
+* [Issue tracker](https://github.com/JetBrains/azure-tools-for-intellij/issues)
+* [Plugin page](https://plugins.jetbrains.com/)
 
-## Data/Telemetry
-Azure Toolkits for Java collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkID=620956) to learn more. If you don't wish to send usage data to Microsoft, you can turn off it in following places:
-* For **Eclipse**: Open `Window > Preferences > Azure`, and uncheck the checkbox.
-* For **IntelliJ IDEA**: Open `Settings/Preferences > Microsoft Tools > Azure`, and uncheck the checkbox.
+## History and differences with Microsoft Azure Toolkits for Java
 
-## Contribution
-please see the [contribution instructions](CONTRIBUTING.md) if you wish to build the plugins from source.
+The Azure plugin for [JetBrains Rider](https://www.jetbrains.com/rider) is a fork of the [Microsoft Azure Toolkits for Java](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation), available [on GitHub](https://github.com/Microsoft/azure-tools-for-java).
+
+Microsoft's Azure Toolkits for Java provide similar functionality to the Azure plugin for [JetBrains Rider](https://www.jetbrains.com/rider), however focus on the Java/JVM ecosystem and development flows. JetBrains [opened a pull request](https://github.com/Microsoft/azure-tools-for-java/pull/1725) to split base functionality (such as browsing Azure resources) from Java/JVM-specific features (such as deploying a `.war` file to the HDInsight service), which would open up the original plugin for various other ecosystems.
+
+While waiting for this PR to be accepted, we are releasing the forked Azure plugin for [JetBrains Rider](https://www.jetbrains.com/rider), with several notable differences:
+
+* No telemetry or usage data is collected and sent to Microsoft
+* Icons have been replaced by custom icons, as [the original icons are not open-source](https://github.com/Microsoft/azure-tools-for-java/issues/1626)
+* Java/JVM-specific functionality was removed
+* .NET-specific functionality, such as deploying an ASP.NET Core web application, has been added
+
+## Contributing
+
+Please see the [contribution instructions](CONTRIBUTING.md) if you wish to build the plugin from source.
 
 ## Disclaimer
-*azure-tools-for-java uses JxBrowser http://www.teamdev.com/jxbrowser, which is a proprietary software. The use of JxBrowser is governed by JxBrowser Product Licence Agreement http://www.teamdev.com/jxbrowser-licence-agreement. If you would like to use JxBrowser in your development, please contact TeamDev.*
+
+*azure-tools-for-java uses JxBrowser http://www.teamdev.com/jxbrowser, which is proprietary software. The use of JxBrowser is governed by JxBrowser Product Licence Agreement http://www.teamdev.com/jxbrowser-licence-agreement. If you would like to use JxBrowser in your development, please contact TeamDev.*
```

Comment:
Checking if we can drop that... build s running... :)

Reply:
@IvanPashchenko just merged that branch. We have dropped this dependency. :yay:

Reply:
Dropped